### PR TITLE
change newsletter text

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

File: _includes/main.liquid [hunk 1]
```diff
@@ -118,15 +118,15 @@
...
+                    <h1 class="h1 cta-subscribe-h1">Subscribe to the latest Chainlink Developer news</h1>
                 </div>
                 <div class="form-subscribe-wrapper w-form">
                     <form id="wf-form-Chainlink-Newsletter" name="wf-form-Chainlink-Newsletter" data-name="Chainlink Newsletter" class="form-subscribe">
                         <div class="form-subscribe-field-wrapper"><label for="Email" class="field-label-hidden">Email Address</label><input type="email" class="cta-subscribe-input w-input" maxlength="256" name="Email" data-name="Email" placeholder="Enter your email address" id="Email" required=""><input id="subscribe-button" type="submit" value="Sign up" class="btn btn-secondary w-button"></div>
                     </form>
                     <div class="form-success-message w-form-done">
                         <div class="subscribe-success-message-text">Thank you for signing up! Please check your inbox to confirm your subscription.</div>
-                            <div class="subscribe-success-social">
+                            <div class="subscribe-success-social
```
Comment (PatrickAlphaC) [inline _includes/main.liquid:129]: Add back

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
Just pass the CI and then looks good!

File: _includes/main.liquid (hunk 1 above)
Comment (StephenFluin) [reply _includes/main.liquid]: The whole HTML was invalid previously. Fixed.

**Comment by StephenFluin** [comment]
@PatrickAlphaC How is it looking?

**Review by PatrickAlphaC — APPROVED** [review]

**Comment by PatrickAlphaC** [comment]
LGTM!

**Pull request merged**